components/tests/python/library/__init__.py

```diff
@@ -214,6 +214,12 @@ def new_image(self, name=""):
         img.acquisitionDate = rtime(0)
         return img
 
+    def new_dataset(self, name="", description=None):
```

Comment:
For better performance? With creating 2 objects, linking them and saving only link you make 1 db call rather then 3, right? In the future we could have one helper new_iobject(class, name) that allows you to create all type of subclass of ObjectI and one make_object(class, name) to create and save.